docs/cuda-target.md

```diff
@@ -153,7 +153,7 @@ RWTexture2d<float2> g_texture;
 
 float2 getValue(RWTexture2D<float2> t)
 {
-    return t[int2(0, 0];
+    return t[int2(0, 0)];
```

Comment:
This and line 170 is a nontrivial change. Please verify my assumption that end parentheses are necessary.